src/patch-babel-types.coffee

```diff
@@ -0,0 +1,40 @@
+require 'babel-types'
+
+{
+  default: defineType
+  assertNodeType
+  assertValueType
+  chain
+  assertEach
+  VISITOR_KEYS
+} = require 'babel-types/lib/definitions'
+
+module.exports = ->
+  return if VISITOR_KEYS.JSXFragment
+
+  defineType 'JSXFragment',
```

Comment:
These are copied from the relevant `babel-types` definitions

This general approach is mimicking what I did on [`es2coffee`](https://github.com/helixbass/es2coffee/blob/master/src/override-babel-types.coffee) for patching `babel-types`